src/main/java/com/mitchej123/hodgepodge/Hodgepodge.java

```diff
@@ -72,6 +78,9 @@ public void preinit(FMLPreInitializationEvent event) {
     public void init(FMLInitializationEvent event) {
         FMLCommonHandler.instance().bus().register(ANCHOR_ALARM);
         NetworkHandler.init();
+        if (Compat.isGT5Present()) {
+            BetterCrashesCompat.init();
+        }
```

Comment:
Meant here. Why GT5u check?

Reply:
if anything shouldnt it be either bettercrashes or Angelica?
Could also use a side check

Reply:
Lol wtf.  Auto complete fail...